### PR TITLE
Remove : from commands names

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed) https://github.com/rizinorg/book/pull/39

**Detailed description**

In the future we might want to use `:` as a delimiter for command "specifiers" (e.g. `pd:table`, `afb:json`, etc.), thus having `:` as part of the command name makes that move impossible.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by XVilka** [comment]
Don't forget please to send PR to the book

File: librz/core/cmd_descs/cmd_seek.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ commands:
       - name: addr
         type: RZ_CMD_ARG_TYPE_RZNUM
         optional: true
-  - name: "s:"
+  - name: s0
```
Comment (XVilka) [inline librz/core/cmd_descs/cmd_seek.yaml]: What about more meaningful name? `spad` or something like that?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/core/cmd_descs/cmd_seek.yaml (hunk 1 above)
Comment (ret2libc) [reply librz/core/cmd_descs/cmd_seek.yaml]: ok!

Comment (ret2libc) [reply librz/core/cmd_descs/cmd_seek.yaml]: Updated

**Review by XVilka — APPROVED** [review]

**Comment by ret2libc** [comment]
I will merge this after I update the book. Otherwise I will forget the book :D

**Pull request merged**